js/client-mainmenu.js

```diff
@@ -379,11 +379,18 @@
 			var $searchForm = $('.mainmenu button.big').closest('form');
 			var $formatButton = $searchForm.find('button[name=format]');
 			var $teamButton = $searchForm.find('button[name=team]');
-			if (this.searching) {
+			if (this.searching && (!$.isArray(this.searching) || this.searching.length)) {
```

Comment:
That `this.searching` is a plain object (like JSON data sent in current protocol), or an array with one or more elements (like JSON data sent in protocol proposed in https://github.com/Zarel/Pokemon-Showdown/pull/1910).